sagemaker-fundamentals/create-training-job/container_kms/train.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+import boto3
+    
+import base64
+from cryptography.fernet import Fernet
```

Comment:
Use the AWS Encryption SDK, it is a better primitive for this use case

Reply:
The idea of notebooks in `/sagemaker-fundamentals` is to provide low-level understanding of AWS components used for a SageMaker workflow. So making code shorter and "simpler" is not the primary objective in `/sagemaker-fundamentals`. The primary objective is to make the code long enough so that all ideas are demonstrated.

Reply:
The primary objective is show how SageMaker workflow works, not to demonstrate how to encrypt things. All you need is `encrypt` and `decrypt`. Trying to get more length by making these function is just offering opportunities for confusion without adding any value to the point of the example :)